source/module_lr/utils/lr_util.cpp

```diff
@@ -101,88 +105,100 @@ namespace LR_Util
     container::Tensor mat2ten_double(ModuleBase::matrix& m)
     {
         container::Tensor t(DAT::DT_DOUBLE, DEV::CpuDevice, { m.nr, m.nc });
-        for (int i = 0;i < t.NumElements();++i) {t.data<double>()[i] = m.c[i];
-}
+        for (int i = 0;i < t.NumElements();++i) {            
+t.data<double>()[i] = m.c[i];
+        }
         return t;
     }
     std::vector<container::Tensor> mat2ten_double(std::vector<ModuleBase::matrix>& m)
     {
         std::vector<container::Tensor> t;
-        for (int i = 0;i < m.size();++i) { t.push_back(mat2ten_double(m[i]));
-}
+        for (int i = 0;i < m.size();++i) {
+            t.push_back(mat2ten_double(m[i]));
+        }
         return t;
     }
     ModuleBase::matrix ten2mat_double(container::Tensor& t)
     {
         ModuleBase::matrix m(t.shape().dims()[0], t.shape().dims()[1]);
-        for (int i = 0;i < t.NumElements();++i) {m.c[i] = t.data<double>()[i];
-}
+        for (int i = 0;i < t.NumElements();++i) {
+            m.c[i] = t.data<double>()[i];
+        }
         return m;
     }
     std::vector<ModuleBase::matrix> ten2mat_double(std::vector<container::Tensor>& t)
     {
         std::vector<ModuleBase::matrix> m;
-        for (int i = 0;i < t.size();++i) { m.push_back(ten2mat_double(t[i]));
-}
+        for (int i = 0;i < t.size();++i) {
+            m.push_back(ten2mat_double(t[i]));
+        }
         return m;
     }
     container::Tensor mat2ten_complex(ModuleBase::ComplexMatrix& m)
     {
         container::Tensor t(DAT::DT_COMPLEX_DOUBLE, DEV::CpuDevice, { m.nr, m.nc });
-        for (int i = 0;i < t.NumElements();++i) {t.data<std::complex<double>>()[i] = m.c[i];
-}
+        for (int i = 0;i < t.NumElements();++i) {
+            t.data<std::complex<double>>()[i] = m.c[i];
+        }
         return t;
     }
     std::vector<container::Tensor> mat2ten_complex(std::vector<ModuleBase::ComplexMatrix>& m)
     {
         std::vector<container::Tensor> t;
-        for (int i = 0;i < m.size();++i) { t.push_back(mat2ten_complex(m[i]));
-}
+        for (int i = 0;i < m.size();++i) {
+            t.push_back(mat2ten_complex(m[i]));
+        }
         return t;
     }
     ModuleBase::ComplexMatrix ten2mat_complex(container::Tensor& t)
     {
         ModuleBase::ComplexMatrix m(t.shape().dims()[0], t.shape().dims()[1]);
-        for (int i = 0;i < t.NumElements();++i) {m.c[i] = t.data<std::complex<double>>()[i];
-}
+        for (int i = 0;i < t.NumElements();++i) {
+            m.c[i] = t.data<std::complex<double>>()[i];
+        }
         return m;
     }
     std::vector<ModuleBase::ComplexMatrix> ten2mat_complex(std::vector<container::Tensor>& t)
     {
         std::vector<ModuleBase::ComplexMatrix> m;
-        for (int i = 0;i < t.size();++i) { m.push_back(ten2mat_complex(t[i]));
-}
+        for (int i = 0;i < t.size();++i) {
+            m.push_back(ten2mat_complex(t[i]));
+        }
         return m;
     }
```

Comment:
these functions seem to be interfaces of Tensor, should be implemented there, you can ask advice from @denghuilu about how to support such interfaecs.